src/quantity.rs

```diff
@@ -354,12 +354,12 @@ macro_rules! quantity {
             /// Creates a struct that formats `self` for display.
             ///
             /// # Notes
-            /// Unlike [`format_args`](#method.format_args), the return value of this method can be
+            /// Unlike [`format_args`](Self::format_args()), the return value of this method can be
```

Comment:
These methods are included in the documentation for both `Quantity` (e.g. ) and individual `Quantity` type aliases (e.g. https://docs.rs/uom/latest/uom/si/absement/type.Absement.html#method.into_format_args). The new item links work for the `Quantity` page but on the type alias pages redirect to `Quantity` instead of the alias's documentation. Is there any way to keep these consistent without using the old style HTML links?

Reply:
Interesting! I didn't spot that.

It doesn't seem possible. Even `[`format_args`](Action::format_args())` resolves to `struct.Quantity.html#method.format_args`, which does make sense because Action is a type alias for Quantity.

Do you want me to change it back to using the original HTML link?

Reply:
Yeah, go ahead and change back.

Reply:
OK, I changed those back and checked that it links as expected.